index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const webidlWrapper = require('./webidl2js-wrapper.js');
+const webidlConversions = require('webidl-conversions');
+
+const sharedGlobalObject = {};
+webidlWrapper.install(sharedGlobalObject);
+
+const origCSSStyleDeclaration = sharedGlobalObject.CSSStyleDeclaration;
+
+/**
+ * @constructor
+ * @param {((cssText: string) => void) | null} [onChangeCallback]
+ * The callback that is invoked whenever a property changes.
+ */
+function CSSStyleDeclaration(onChangeCallback = null) {
```

Comment:
I was trying to say to use `undefined` instead of `null` as the sentinel for the absence of a callback.

```suggestion
function CSSStyleDeclaration(onChangeCallback = undefined) {
```

Reply:
Well, the existing version of the constructor supports passing `null`.